src/mnesia_api.erl

```diff
@@ -174,7 +175,7 @@ install_fallback_mnesia(Path) ->
             {cannot_fallback, String}
     end.
 
--spec set_master(Node :: atom() | string()) -> {error, io_lib:chars()} | {ok, []}.
+-spec set_master(Node :: atom() | string()) -> {cannot_set, io_lib:chars()} | {ok, []}.
```

Comment:
```suggestion
-spec set_master(node()) -> {cannot_set, io_lib:chars()} | {ok, []}.
```

Remove lines 179-182, because we always have atoms here, right?